docs/src/api/class-browsercontext.md

```diff
@@ -774,6 +903,14 @@ page.goto("https://example.com")
 browser.close()
 ```
 
+```csharp
+var context = await browser.NewContextAsync();
+var page = await context.NewPageAsync();
+await context.RouteAsync(new System.Text.RegularExpressions.Regex("(\\.png$)|(\\.jpg$)"), r => r.AbortAsync());
```

Comment:
How about?

```suggestion
await context.RouteAsync(new Regex("(\\.png$)|(\\.jpg$)"), r => r.AbortAsync());
```

Reply:
Sure, I kept it compilable without `using` :) Works for me though.